LPChoose.jl

```diff
@@ -1,12 +1,17 @@
 #include("LPChoose.jl"); LPChoose("smalldata.txt",100,0.00);
 using DataFrames,LinearAlgebra,CSV,DelimitedFiles,SparseArrays,GLPK,JuMP,Statistics, StatsBase
 using ProgressMeter
+
+import_Gurobi = false  # import Gurobi package when import_Gurobi = true
+if import_Gurobi == true
+    using Gurobi
```

Comment:
```
try
    @eval import Gurobi
    println("Gurobi has been installed.")
catch
    println("Gurobi is not installed. GLPK has been installed.")
end
```

Reply:
consider a better printout here to give users some suggestion which solver to use.